### ISSUE TITLE
Please ease the pain of many WSL users here

### ISSUE THREAD

**Issue by benqzq** [Microsoft/WSL#2961]
Please see this:

https://unix.stackexchange.com/questions/424966/solve-all-trivial-character-problems-when-doing-ubuntu-bash-from-windows-10-with

Yes, I know it will be closed after 5 minutes but I strongly suggest all WSL developers to take this pain of many users seriously, because I and many others really ate straw with these issues... And I really like WSL and want to keep using the good WSL-Putty stack.

**Comment by lewisdonofrio** [comment]
I agree with this - VcXsrv  I run full gui desktop xfce4 for my daily driver (www.tinyurl.com/donofrioworkdesk fwiw ignore unneeded onedrive prompt) I only edit files with /mnt/c/Users/username have not run into dos4unix in a decade, was cygwin full desktop before wsl made my life so much better.

**Comment by Biswa96** [comment]
@benqzq You mentioned a question that is too broad. Be specific. And MS only developing the CLI part. GUI is way behind. WSL is mainly for developers and they like commands.

**Comment by benqzq** [comment]
I humbly consider myself a junior developer and I like commands two. AFAIK these problems relate to nearly any WSL user who establishes common web environments (LAMP/LEMP), yet I know no one has to agree with this opinion.

**Comment by benhillis** [comment]
@benqzq - I'm unsure what you are asking for. Could you please elaborate?

**Comment by therealkenc** [comment]
The linked post appears to have been removed from Stack Exchange. It was there three days ago being down-voted last I looked.
